fw/http_limits.c

```diff
@@ -361,7 +357,8 @@ tfw_classify_conn_close(struct sock *sk)
 
 	spin_unlock(&ra->lock);
 
-	sk->sk_security = NULL;
+	if(sk)
```

Comment:
sk is always not zero here!

Reply:
https://github.com/tempesta-tech/linux-5.10.35-tfw/pull/25/files#r1582808517

Reply:
I think he means BUG_ON(!ra);  ra is saved in sk_security